spec/models/search/response_search_spec.rb

```diff
@@ -74,7 +74,7 @@
           [1, "fox heaven jumps", "bravo", "fuzzy gusher", "apple heaven ipswitch", "Dog", %w[hammer wrench]])
       end
       let!(:r3) do
-        create(:response, form: form, reviewed: true, answer_values:
+        create(:response, form: form, reviewed: false, answer_values:
```

Comment:
Why did this get changed?

Reply:
debugging; forgot to switch

Thanks for the catch